test/src/e2e.long/reward2.test.ts

```diff
@@ -97,7 +97,13 @@ describe("reward2", function() {
 
         // nodeA creates 2 blocks
         {
-            await nodeA.pay(aliceAddress, 100); // +160 for alice in nodeA
+            await nodeA.pay(aliceAddress, 100); // +100 +50 +10*2/10 for alice in nodeA, +10*1/10 for bob
```

Comment:
I think the test would have been easier if we used a scheme without `engine.solo.params.genesisStakes`. Can we split the scheme?

Reply:
Splitting the scheme is okay, but even though we should test it too.
The e2e test should be a real scenario, not an easy-to-test scenario.